app/services/irs_attempts_api/redis_client.rb

```diff
@@ -30,8 +30,8 @@ def read_events(count = 1000)
     end
 
     def self.clear_attempts!
-      if !Rails.env.test?
-        raise 'IrsAttemptsApi::RedisClient.clear_attempts! should not be called outside of test env'
+      unless %w(test development).include?(Rails.env)
+        raise 'IrsAttemptsApi::RedisClient.clear_attempts! should not be called outside of dev or test!s'
```

Comment:
```suggestion
        raise 'IrsAttemptsApi::RedisClient.clear_attempts! should not be called outside of dev or test!'
```

Think I fat-fingered Ctrl+S here.